crates/cubecl-linalg/src/matmul/kernels/tiling2d/tile/loader.rs

```diff
@@ -52,7 +52,7 @@ impl<N: Numeric> Loader<N> for TileLoader<N> {
     ) {
         let dims = load_info.dims;
         let coordinates = load_info.coordinates;
-        let gm_stride = dims.m;
+        let gm_stride = lhs.stride(lhs.rank() - 1);
```

Comment:
Same here. Is this still correct? And why the change?

Reply:
This one wasn't even correct before, since `matrix_layout` never checked that the input was contiguous, only ordered. So a sliced input tensor would yield incorrect results. For transposed tensors, the `k` stride is the last one. We check that in `matrix_layout`, so the assumption is safe.